tests/test_target/test_binning.py

```diff
@@ -0,0 +1,40 @@
+# tests.test_target.test_binning
+# Tests for the BalancedBinningReference visualizer
+#
+# Author:   Juan L. Kehoe (juanluo2008@gmail.com)
+# Author:  Prema Damodaran Roman (pdamo24@gmail.com)
+# Created: Thu Jul 20 10:21:49 2018 -0400
+#
+# ID: test_binning.py
+
+import pytest
```

Comment:
This statement was causing the [flakes check to fail with Unused Import error](https://travis-ci.com/DistrictDataLabs/yellowbrick/jobs/136370433#L1865). If you remove `import pytest` it should work.